core/server/prometheus-metrics.js

```diff
@@ -3,25 +3,31 @@
 const prometheus = require('prom-client')
 
 module.exports = class PrometheusMetrics {
```

Comment:
so beforehand, this class was responsible both for managing metrics and determining whether it was configured to do so? now that configuration/enablement responsibility has shifted, and this class just does its metrics thing right?

seems like a reasonable change if so 👍

Reply:
Yea, there used to be more config but it was removed, and now that it has been, that responsibility is better handled elsewhere.